custom_components/anniversaries/sensor.py

```diff
@@ -1,24 +1,18 @@
 """ sensor """
 
 from dateutil.relativedelta import relativedelta
-from datetime import datetime, date, timedelta

```

Comment:
This is fine, timedelta is no longer used since switching to relativedelta

Reply:
👍